library/src/main/java/com/wdullaer/materialdatetimepicker/date/DayPickerView.java

```diff
@@ -417,6 +417,11 @@ public boolean performAccessibilityAction(int action, Bundle arguments) {
             }
         }
 
+        // Only attempt to scroll to another month if it is in range
+        if (!canGoTo(day)) {
```

Comment:
So this looks like it'll work, but I think we can simplify the entire function.
This accessibility code still dates from when the DayPickerView was a ListView rather than a RecyclerView.

In fact these two accessibility events need to have the same effect as the arrow buttons in the DayPickerGroup.
Over there I'm just manipulating the RecyclerView position rather than a Date, which is simpler IMO.

See: https://github.com/wdullaer/MaterialDateTimePicker/blob/cc53bb190046b301a1ff31f320b69c742ec9ec2a/library/src/main/java/com/wdullaer/materialdatetimepicker/date/DayPickerGroup.java#L168

What do you think?